src/components/ThreadEnvelope.vue

```diff
@@ -71,16 +71,18 @@
 			</router-link>
 			<div class="right">
 				<Moment class="timestamp" :timestamp="envelope.dateInt" />
-				<button
-					:class="{
-						'icon-reply-all-white': hasMultipleRecipients,
-						'icon-reply-white': !hasMultipleRecipients,
-						primary: expanded,
-					}"
-					class="button"
+				<Button
```

Comment:
did you make a screenshot of your Nextcloud 22 test by any chance? I wonder how the button renders

Reply:
yes, check the description